registry/src/skills.json

```diff
@@ -7,6 +7,72 @@
         "author": null,
         "description": "Enables your agent to fetch token details for any token on Base",
         "githubUrl": "https://github.com/moxie-protocol/moxie-agent-skills/tree/main/packages/plugin-moxie-token-details",
+        "imageUrl": "https://github.com/moxie-protocol/moxie-agent-skills/tree/main/packages/plugin-moxie-token-details/images/logo.png"
+    },
+    {
+        "pluginId": "ba2e0746-d089-4539-a23b-84882bc8eafd",
+        "name": "@moxie-protocol/plugin-betswirl",
+        "version": "0.0.1",
+        "displayName": "Play Games of Luck",
+        "author": "Romuald Hog <romuald@betswirl.com>",
+        "description": "Enabled your agent to place bets on BetSwirl's games of luck, and retrieving bets information.",
+        "githubUrl": "https://github.com/moxie-protocol/moxie-agent-skills/tree/main/packages/plugin-betswirl",
+        "logoUrl": "https://github.com/moxie-protocol/moxie-agent-skills/tree/main/packages/plugin-betswirl/images/logo.png",
+        "settings": {},
+        "capabilities": [
+            "Flip a Coin",
+            "Roll a Dice",
+            "Spin a Roulette",
+            "Get your bets"
+        ],
+        "starterQuestions": [
+            {
+                "label": "Bet 0.01 ETH on heads",
+                "value": "Bet 0.01 ETH on heads"
+            },
+            {
+                "label": "Bet 0.01 ETH above 44",
+                "value": "Bet 0.01 ETH above 44"
+            },
+            {
+                "label": "Bet 0.01 ETH on 3,6,8 and 10",
+                "value": "Bet 0.01 ETH on 3,6,8 and 10"
+            },
+            {
+                "label": "Show my bets",
+                "value": "Show my bets"
+            }
+        ],
+        "mediaUrls": [
+            "https://github.com/moxie-protocol/moxie-agent-skills/tree/main/packages/plugin-betswirl/images/logo.png"
+        ],
+        "actions": [
+            "COIN_TOSS",
+            "COIN_FLIP",
+            "DOUBLE_OR_NOTHING",
+            "TOSS_A_COIN",
+            "BETSWIRL_COIN_TOSS",
+            "DICE",
+            "ROLL_A_DICE",
+            "DICE_ROLL",
+            "BETSWIRL_DICE",
+            "ROULETTE",
+            "SPIN_A_ROULETTE",
+            "BETSWIRL_ROULETTE",
+            "GET_BETS",
+            "RETRIEVE_BETS",
+            "SHOW_BETS",
+            "LAST_BETS",
+            "GET_BETSWIRL_BETS"
+        ],
+        "isPremium": false,
+        "freeQueries": 0,
+        "skillCoinAddress": "",
+        "minimumSkillBalance": 0,
+        "status": "ACTIVE",
+        "isDefault": false
+    },
```

Comment:
Other than that, there's also new fields `loaders` which you can customize the loader text when loading the response as shown in the screen here:

<img width="551" alt="Screenshot 2025-04-02 at 22 29 43" src="https://github.com/user-attachments/assets/8a1a2416-240a-4551-8888-56a8e4cf37ed" />

Simply just add like this:
```suggestion
        "loaders": [
            "Text 1",
            "Text 2"
        ]
    },
```

This is optional, so if you don't want to, feel free to skip it